### PR TITLE
[PW-4823] prefill shopper details and addresses 

### PR DESCRIPTION
**Pull request by peterojo**
## Summary
Prefill personal details and addresses in components that require this data

## Tested scenarios

Afterpay
Oney (3x, 4x)

**Fixed issue**:

### CODE REVIEW

**Review by acampos1916 — APPROVED** [review]

**Review by Morerice — APPROVED** [review]

File: src/Resources/app/storefront/src/configuration/adyen.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default {
         'facilypay_4x', 'facilypay_6x', 'facilypay_10x', 'facilypay_12x', 'afterpay_default', 'ratepay',
         'ratepay_directdebit', 'giftcard'
     ],
+    prefillablePaymentMethods: ['facilypay_3x', 'facilypay_4x', 'facilypay_6x', 'facilypay_10x', 'facilypay_12x', 'afterpay_default'],
```
Comment (cyattilakiss) [inline src/Resources/app/storefront/src/configuration/adyen.js:29]: it is possible to just add it to the main component as well so all the methods get this configuration or is it better to keep it separated per method?

Comment (peterojo) [reply src/Resources/app/storefront/src/configuration/adyen.js:29]: I think not all the PMs accept the fields `personalDetails`,`billingAddress` and `deliveryAddress`.  At least going by the fields present in their `details` array in the `/paymentMethods` response. To be honest I don't know what happens if I just set this data for all PMs.

Comment (cyattilakiss) [reply src/Resources/app/storefront/src/configuration/adyen.js:29]: This is what we do in other plugins as a recommendation from checkout, they are going to take care of filtering out the not necessary fields in the data config, so this should make our life easier regarding maintenance :)

Comment (peterojo) [reply src/Resources/app/storefront/src/configuration/adyen.js]: Updated to add the addresses for all components, still works as well as the others

**Review by cyattilakiss — APPROVED** [review]

**Review by Morerice — APPROVED** [review]

**Pull request merged**